packages/cli/__tests__/zosfiles/__integration__/create/ds/__scripts__/create_ds_help.sh

```diff
@@ -2,7 +2,7 @@
 set -e
 
 echo "================Z/OS FILES CREATE DATA-SET HELP==============="
-zowe files create data-set --help
+zowe files create --help
```

Comment:
Wondering if this was intended.
Since this script belongs to the `.../create/ds` folder, shouldn't it display data-set help?

Reply:
I was getting an error on the integration tests...I can try to remove it again and see if they pass now